relay-server/src/services/store.rs

```diff
@@ -972,6 +910,121 @@ impl StoreService {
 
         Ok(())
     }
+
+    fn produce_metrics_summary(
+        &self,
+        scoping: Scoping,
+        start_time: Instant,
+        item: &Item,
+        span: &SpanKafkaMessage,
+    ) {
+        let payload = item.payload();
+        let d = &mut Deserializer::from_slice(&payload);
+        let mut metrics_summary: SpanWithMetricsSummary = match serde_path_to_error::deserialize(d)
+        {
+            Ok(span) => span,
+            Err(error) => {
+                relay_log::error!(
+                    error = &error as &dyn std::error::Error,
+                    "failed to parse span"
+                );
+                self.outcome_aggregator.send(TrackOutcome {
+                    category: DataCategory::SpanIndexed,
+                    event_id: None,
+                    outcome: Outcome::Invalid(DiscardReason::InvalidSpan),
+                    quantity: 1,
+                    remote_addr: None,
+                    scoping,
+                    timestamp: instant_to_date_time(start_time),
+                });
```

Comment:
We should not emit an outcome here (at least not for spans), because at this point we have already parsed a `SpanKafkaMessage` successfully and there will be an accepted outcome for it in `produce_span`.

```suggestion
```